### DISCUSSION TITLE
[Feature]: 添加双指触摸两次的快捷键

### DISCUSSION THREAD

**Discussion by UlinoyaPed** [immersive-translate/immersive-translate#985] [Ideas]
### 新功能 | New Feature

在界面设置-键盘快捷键中添加**双指触摸两次**唤起翻译的快捷键设置

### 为什么您想要添加这些功能 | Why you want to add these features

这还要从另一个翻译插件说起
[TWP](https://github.com/FilipePS/Traduzir-paginas-web) 这个翻译插件我一直在用，其中唤起翻译面板就设置的是三指触摸，现在我想两个插件一起使用但**三指触摸时会同时触发两指的操作**，这使得两个插件打架，希望添加对触摸操作的监测和双指触摸两次的支持。

**Comment by theowenyoung** [comment]
触摸2次 误触的几率比较大. 如果你给沉浸式翻译设置的2指触摸的话，那么 似乎 TWP 不应该被触发，那么这个issue应该提给 twp？

或者直接设置 4指触发。
